src/main/java/com/ververica/flink/table/gateway/context/ExecutionContext.java

```diff
@@ -163,18 +149,11 @@ private ExecutionContext(
 			environment.getDeployment(),
 			commandLineOptions);
 
-		flinkConfig.addAll(createExecutionConfig(
+		this.flinkConfig.addAll(createExecutionConfig(
```

Comment:
`this.flinkConfig` and `flinkConfig` are the same.

Reply:
yes, I think `this.flinkConfig` is more clear